bpf/test/bpf_lseek.c

```diff
@@ -40,12 +40,15 @@ char _license[] __attribute__((section(("license")), used)) = "GPL";
 __attribute__((section(("tracepoint/sys_enter_lseek")), used)) int
 test_lseek(struct sys_enter_lseek_args *ctx)
 {
+	// NB: this values should match BogusFd and  BogusWhenceVal in
+	// pkg/sensrors/test
 	if (ctx->fd == -1 && ctx->whence == 4444) {
```

Comment:
We might consider using a more baroque whence here so that it's harder to accidentally write a test in the future that clashes with this value. I can imagine that being the source of some very confusing test bugs.

Reply:
OK, used 4729.